.github/workflows/gccgo.yml

```diff
@@ -0,0 +1,14 @@
+on: [push, pull_request]
+jobs:
+  gccgo:
+    runs-on: ubuntu-latest
+    name: "gccgo Build"
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install build utils
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y gcc-multilib gccgo
+      - run: go-10 version
```

Comment:
What is `go-10`?

Reply:
In order to avoid conflict between default go compiler and gcc go compiler, gccgo package from Ubuntu has a suffix of corresponding gcc version, and it is `10` in this case meaning it comes with gcc 10. AFAIK, Debian uses the same convention, whereas Arch disallows  gccgo and default go compiler to be installed at the same time.

Reply:
However gcc 10 cannot compile go 1.15 properly, so we may need upstream GitHub base image to update.

Reply:
Makes sense. The Go versions available via `apt` are always unpredictable and usually outdated.
Can we download the code from golang.org?